src/hearings/containers/request-hearing/request-hearing.component.ts

```diff
@@ -11,6 +11,7 @@ import { AbstractPageFlow } from '../../utils/abstract-page-flow';
   styleUrls: ['./request-hearing.component.scss']
 })
 export class RequestHearingComponent implements OnDestroy {
+  isButtonSubmit: boolean = false;
```

Comment:
Can we call this variable "isSubmitInProgress" - current name quite confusing

Reply:
changed meaningfull variable name from isButtonSubmit to isSubmitInProgress